### DISCUSSION TITLE
How to Print GUI Version?

### DISCUSSION THREAD

**Discussion by Bwanna** [ImpulseAdventure/GUIslice#479] [Q&A]
I realize this is a super dumb question, but how do you serial print the GUIslice Version?
How to translate the .h file information into a Serial.print(); statement?

I'm assuming there is a variable needed in the parens, but can't find what this might be.
Serial.print(gslc_GetVer());

```
[GUIslice.h]
// Get the GUIslice version number
//
// \param[in]  pGui:      Pointer to GUI
//
// \return String containing version number
//
char* gslc_GetVer(gslc_tsGui* pGui);
```

**Comment by Pconti31** [comment]
@Bwanna I think the easiest way is to add the following include at the top of your *.ino file then simply do:
```
#include "GUIslice_version.h"

Serial.println(GUISLICE_VER);
```
or to use:
```
Serial.println(gslc_GetVer(&m_gui));
```

Paul--

**Reply by Bwanna** [reply]
Thank you. This helps!
